python/src/aiconfig/scripts/aiconfig_cli.py

```diff
@@ -46,14 +50,33 @@ def run_subcommand(argv: list[str]) -> Result[str, str]:
         LOGGER.debug("Running edit subcommand")
         res_edit_config = core_utils.parse_args(main_parser, argv[1:], EditServerConfig)
         LOGGER.debug(f"{res_edit_config.is_ok()=}")
-        res_servers = res_edit_config.and_then(_run_editor_servers)
         out: Result[str, str] = result.do(
             #
             Ok(",".join(res_servers_ok))
             #
-            for res_servers_ok in res_servers
+            for edit_config in res_edit_config
+            for cli_config in res_cli_config
+            for res_servers_ok in _run_editor_servers(edit_config, cli_config.config_path)
         )
         return out
+    elif subparser_name == "rage":
+        LOGGER.debug("Running rage subcommand")
+        print("Raging...")
+        rage.spin(2)
+        print("Please hold. Your call is important to us.\nA representative will be with you shortly.")
+        rage.spin(3, type="music")
+        print("Turning up the heat...")
+        rage.spin(2)
+        print("Looking for your flipping server logs...")
+        rage.spin(4, type="music")
+        print("Do you know how many Zuckerbucks I should be charging for this?")
+        rage.spin(5)
+        print("I'm glad we're finally spending time together.")
+        rage.spin(4, type="music")
```

Comment:
Does this end up repeating after we get to the end?

Reply:
Repeating what?

Reply:
Ohhhhhh, so we're just like spending 20s trolling the people who are upset and want to rage shake? We're not actually "waiting" on anything? lmao ok uh.... hmmmmmmmm, I don't know if that's the best user experience. Either way I think we can remove if we really want to and just get this unblocked for now

Reply:
Counterintuitively, I actually think this might be better UX than just doing it. I really think `rage` might be mostly bedside manner and a little engineering.